### PR TITLE
Skip non-certificate PEM blocks

### PR DESCRIPTION
**Pull request by csstaub**
r: @alokmenghrajani @mcpherrinm @christodenny 
Skip non-certificate PEM blocks

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -232,6 +232,10 @@ func readCertsFromFile(wg *sync.WaitGroup, reader io.Reader, filename string, fo
 		scanner := pemScanner(reader)
 		for scanner.Scan() {
 			block, _ := pem.Decode(scanner.Bytes())
+			if block.Type != "CERTIFICATE" {
+				// Skip non-certificate PEM blocks
+				continue
+			}
 			cert, err := x509.ParseCertificate(block.Bytes)
```
Comment (mcpherrinm) [inline main.go]: We should also continue if we fail to parse a cert instead of existing below

Comment (csstaub) [inline main.go]: Disagreed on that point, though I don't feel strongly about it. I want it to stop if the data is corrupted.

**Comment by mcpherrinm** [comment]
🚢

File: main.go (hunk 1 above)
Comment (mcpherrinm) [inline main.go]: Shrug - I was just thinking about a case where you might have something that looks like a PEM certificate but isn't really, and the data you're interested in is below it.  I don't have strong feelings either.

**Pull request merged**